site/layouts/partials/breadcrumb.html

```diff
@@ -2,13 +2,13 @@
 {{ with .Parent }}
     {{ template "breadcrumb" . }}
     {{ if .IsHome }}
-    <a href="{{ .URL }}">
+    <a href="{{ .CurrentSection.Permalink }}">
```

Comment:
Not sure if this PR is the place to fix this, but clicking on "Course Home" brings me to the placeholder index.

Reply:
yeah, I thought that `.CurrentSection.Permalink` might give us the right link but it doesn't yet - I'll fix in a follow-up at some point.